chalicelib/checks/deployment_checks.py

```diff
@@ -217,20 +217,44 @@ def deploy_application_to_beanstalk(connection, **kwargs):
 
 
 @check_function()
-def deploy_ff_staging(connection, **kwargs):
-    """ Deploys Fourfront master to whoever staging is.
-        Runs as part of the 'deployment_checks' schedule on data ONLY.
+def deploy_env(connection, env_to_deploy, name, check, **kwargs):
+    """ For a given beanstalk environment, deploys, or returns an error.
```

Comment:
I would at least say what `name` is in the docstring as it not clear that by `name` you mean `application_name`.